lib/karafka/consumers/callbacks.rb

```diff
@@ -32,22 +41,17 @@ module ClassMethods
       def self.included(consumer_class)
         consumer_class.class_eval do
           extend ClassMethods
-          include ActiveSupport::Callbacks
-
-          # The call method is wrapped with a set of callbacks
-          # We won't run process if any of the callbacks throw abort
-          # @see http://api.rubyonrails.org/classes/ActiveSupport/Callbacks/ClassMethods.html#method-i-get_callbacks
-          TYPES.each { |type| define_callbacks type }
+          include Dry::Events::Publisher[:"karafka_#{self}_event_publisher"]
```

Comment:
should we somehow prettify the `self` here or change this symbol entirely?

It would be useful when we want to subscribe to events from `FooConsumer`, but... without knowing its class name. Doesn't sound like a valid use case.

Reply:
No - we should not subscribe to anonymous events. The only thing that worries me is the fact that it will upcase the self, right?

Reply:
It will, and if the `self` is namespaced the namespace will be here too. If we don't want to subscribe to publishers by magic strings this shouldn't be an issue though.